app/backend/gwells/models/bulk.py

```diff
@@ -51,8 +51,10 @@ class Meta:
     update_to_aquifer = models.ForeignKey(
         Aquifer, db_column='to_aquifer_id', on_delete=models.PROTECT, blank=False,
         null=False, verbose_name='Aquifer ID Number',
+        related_name='update_to_aquifer',
         db_comment=('The aquifer that this well is to be correlated with'))
     update_from_aquifer = models.ForeignKey(
         Aquifer, db_column='from_aquifer_id', on_delete=models.PROTECT, blank=True,
         null=True, verbose_name='Aquifer ID Number',
+        related_name='update_from_aquifer',
```

Comment:
Since this will end up being the name of collections of these objects, the default might make more sense here (`update_from_aquifer_set`), but it's not a big deal.